File: yangxj96-bean/src/main/java/com/yangxj96/saas/bean/system/Route.java
package com.yangxj96.saas.bean.system;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import com.yangxj96.saas.common.base.BaseEntity;
import lombok.*;
import lombok.experimental.SuperBuilder;

import java.io.Serializable;

/**
 * 路由对象数据库实体
 */
@Data
@ToString
@SuperBuilder
@EqualsAndHashCode(callSuper = true)
@AllArgsConstructor
@NoArgsConstructor
@TableName(value = "db_system.t_route")
class Route extends BaseEntity implements Serializable {

    @TableField(value = "uri")
    private String uri;

    @TableField(value = "\"order\"")
    private Integer order;

    @TableField(value = "predicates")
    private String predicates;

    @TableField(value = "filters")
    private String filters;

    @TableField(value = "metadata")
    private String metadata;

    @TableField(value = "route_id")
    private String routeId;

}

File: yangxj96-bean/src/main/java/com/yangxj96/saas/bean/system/Route.kt
/*
 * Copyright (c) 2018 - 2023
 * 作者：杨新杰(Jack Young)
 * 邮箱：yangxj96@gmail.com
 * 博客：www.yangxj96.com
 * 日期：2023-09-13 00:04:11
 * Copyright (c) 2018 - 2023
 */

package com.yangxj96.saas.bean.system

import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableName
import com.yangxj96.saas.common.base.BaseEntity
import java.io.Serializable

@TableName(value = "db_system.t_route")
class Route : BaseEntity(), Serializable {

    @TableField(value = "uri")
    var uri: String? = null

    @TableField(value = "\"order\"")
    var order: Int? = null

    @TableField(value = "predicates")
    var predicates: String? = null

    @TableField(value = "filters")
    var filters: String? = null

    @TableField(value = "metadata")
    var metadata: String? = null

    @TableField(value = "route_id")
    var routeId: String? = null

    override fun toString(): String {
        return "Route(uri=$uri, order=$order, predicates=$predicates, filters=$filters, metadata=$metadata, routeId=$routeId)"
    }

}